xml/System.Formats.Cbor/CborConformanceMode.xml

```diff
@@ -54,7 +62,16 @@
       </ReturnValue>
       <MemberValue>3</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Ensures that the CBOR data is canonical, as specified by the CTAP v2.0 standard, section 6.
+              Extends strict conformance with the following requirements:
+              <list type="bullet">
+                <item>Maps (major type 5) must contain keys sorted by encoding.</item>
+                <item>Indefinite-length items must be made into definite-length items.</item><item>Integers must be encoded as small as possible.</item>
+                <item>The representations of any floating-point values are not changed.</item>
+                <item>CBOR tags (major type 6) are not permitted.</item>
+              </list>
+            </summary>
```

Comment:
Items does not display correctly in the rendered page:
![image](https://user-images.githubusercontent.com/16040868/94736841-ba570000-0321-11eb-8805-66476d8a03b1.png)

Reply:
Nice catch. There are two places using items. I'll investigate what needs to be used.

Reply:
Ok Maira helped me figure this out, and it seems we need a `<description></description>` inside the `<item></item>`, and an optional `<term></term>`.
Example of only a description: https://github.com/dotnet/dotnet-api-docs/blob/3f8449819844e72d42c939b0d551ddca08e85121/xml/Microsoft.Extensions.Hosting/Host.xml#L46

Which looks fine: https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.filesystemglobbing.matcher?view=dotnet-plat-ext-3.1

Or we can include a term: https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.filesystemglobbing.matcher?view=dotnet-plat-ext-3.1

Reply:
```suggestion
        <summary>
          <para>Ensures that the CBOR data is canonical, as specified by the CTAP v2.0 standard, section 6.</para>
          <para>Extends strict conformance with the following requirements:</para>
            <list type="bullet">
              <item><description>Maps (major type 5) must contain keys sorted by encoding.</description></item>
              <item><description>Indefinite-length items must be made into definite-length items.</description></item>
              <item><description>Integers must be encoded as small as possible.</description></item>
              <item><description>The representations of any floating-point values are not changed.</description></item>
              <item><description>CBOR tags (major type 6) are not permitted.</description></item>
            </list>
        </summary>
```